analyticsclient/course.py

```diff
@@ -105,6 +105,16 @@ def problems(self, data_format=DF.JSON):
         path = 'courses/{0}/problems/'.format(self.course_id)
         return self.client.get(path, data_format=data_format)
 
+    def problems_and_tags(self, data_format=DF.JSON):
+        """
+        Get the problems for the course with assigned tags.
+
+        Arguments:
+            data_format (str): Format in which data should be returned
```

Comment:
@nedbat I've copied existed API call to get the list of problems
https://github.com/edx/edx-analytics-data-api-client/blob/master/analyticsclient/course.py#L98
and modified it to use the new server API